docs/fitting_types.rst

```diff
@@ -256,6 +256,37 @@ Let's plot the model for some kinetics constants::
    :width: 300px
    :alt: ODE integration
 
+More common examples, such as dampened harmonic oscillators also work as expected:
+
+  # oscillator strength
+  k = symfit.Parameter()
+  # mass, just there for the physics
+  m = symfit.Parameter(fixed=True)
+  # Dampening factor
+  gamma = symfit.Parameter(1)
+  
+  x, v, a, t = symfit.variables('x, v, a, t')
+  
+  a = (-k * x - gamma * v)/m
+  ode_model = symfit.ODEModel({
```

Comment:
I would prefer to define the `model_dict` separately _and_ in the same style as found elsewhere in the docs.